docs/docs/userguide/PREREQUISITES.md

```diff
@@ -46,19 +47,23 @@ Before installing the Data Center Helm charts you need to set up your environmen
-
-### Configure a shared-home volume
-
-* See examples of [creating shared storage](../examples/storage/STORAGE.md).
-* All of the Data Center products require a shared network filesystem if they are to be operated in multi-node clusters. If no shared filesystem is available, the products can only be operated in single-node configuration.
 
-* The `shared-home` volume must be correctly configured as a read-write shared filesystem (e.g. NFS, AWS EFS, Azure Files)
+!!!info "Reducing Pod to DB latency" 
 
-* The recommended setup is to use Kubernetes PersistentVolumes and PersistentVolumeClaims. The `local-home` volume requires a PersistentVolume with `ReadWriteOnce (RWO)` capability, and `shared-home` requires a PersistentVolume with `ReadWriteMany (RWX)` capability. Typically, this will be a NFS volume provided as part of your infrastructure, but some public-cloud Kubernetes engines provide their own RWX volumes (e.g. AzureFile, ElasticFileStore). 
+      For better performance consider co-locating your database in the Availability Zone (AZ) as your product nodes. Database-heavy operations (e.g. full re-index) become significantly faster when the database is collocated with the Data Center node in the same AZ, however we don't recommend this if you're running critical workloads.
 
+### Configure a shared-home volume
+* All of the Data Center products require a shared network filesystem if they are to be operated in multi-node clusters. If no shared filesystem is available, the products can only be operated in single-node configuration.
+* Some cloud based options for a shared filesystem include [AWS EFS](https://aws.amazon.com/efs/){.external} and [Azure Files](https://docs.microsoft.com/en-us/azure/storage/files/storage-files-introduction){.external}. You can also stand up your own NFS
+* The logical representation of the chosen storage type within Kubernetes can be defined as `PersistentVolumes` with an associated `PersistentVolumeClaims` in a `ReadWriteMany (RWX)` access mode.
 * For more information about volumes go to the [Volumes section of the configuration guide](CONFIGURATION.md#volumes). 
+* See examples of [creating shared storage](../examples/storage/STORAGE.md).
+
+### Configure local-home volume
+* As with the [shared-home](#configure-a-shared-home-volume) each pod requires its own volume for `local-home`. This is needed by each product for defining operational data. 
+* If not defined, an [emptyDir{}](https://kubernetes.io/docs/concepts/storage/volumes/#emptydir){.external} will be utilised. 
+* Whilst an `emptyDir` may be acceptable for evaluation purposes its recommended that each Pod is allocated its own volume
```

Comment:
I am not an expert writer, but I think better use `it is recommended`

Reply:
Not sure either actually, I do know Im missing an apostrophe. Ill update to your recommendation anyway.